### PR TITLE
feat: Add hover feedback

### PR DESCRIPTION
**Pull request by geomod-team**
The pull request :
* Adds hover feedback on the radial menu:
  * It possible to change button background color;
  * It possible to change button text color;
  * It possible to add an hover callback funtion. The idea is for example change a CSS variable (see index.html)
* Adds doc generation. To generate api doc from jsDoc, do : `npm run docs`

### CODE REVIEW

File: src/RadialMenu.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ const NO_SHADOW = { color: 'transparent', blur: 0 };
 /** @type {number} */
 const TWOPI = 2 * Math.PI;
 
-class RadialMenu {
+export default class RadialMenu {
```
Comment (victorqribeiro) [inline src/RadialMenu.js:71]: this is breaking in browser, even if I update the index.html and change to ```<script src="src/RadialMenu.js" type="module"></script>

**Comment by victorqribeiro** [comment]
you also forgot to update index.html - I changed it to use the build version - build/RadialMenu.min.js

I create this build using `uglifyjs src/RadialMenu.js -c -m -o build/RadialMenu.min.js`

**Review by victorqribeiro — CHANGES_REQUESTED** [review]

File: src/RadialMenu.js (hunk 1 above)
Comment (geomod-team) [reply src/RadialMenu.js:71]: I had `export default` to use esModule. It is just an idea.

To run index.html in local we have to install and run server like [Web Server for Chrome](https://chrome.google.com/webstore/detail/web-server-for-chrome/ofhbbkphhbklhfoeikjpcbhemlocgigb).  
Run html file in browser with file:// is not possible with modules [see here](https://medium.com/ghostcoder/using-es6-modules-in-the-browser-5dce9ca9e911)

Comment (victorqribeiro) [reply src/RadialMenu.js:71]: yes, I have a server running locally. it was just an lack of update in index.html. Could you please also update style-visualizer.html and the README.md to reflect your changes before I merge?

Comment (geomod-team) [reply src/RadialMenu.js:71]: I update style-visualizer.html. 

The README file is up to date. :-) 

I hope that's ok.

**Comment by victorqribeiro** [comment]
Thank you.

**Comment by victorqribeiro** [comment]
I think it would be nice if you could add a new style using the features you've add, in the style visualizer. What do you think?

**Pull request merged**